app/assets/javascripts/admin/adapters/web-hook-event.js.es6

```diff
@@ -0,0 +1,7 @@
+import RESTAdapter from 'discourse/adapters/rest';
+
+export default RESTAdapter.extend({
+  basePath() {
+    return '/admin/api/';
```

Comment:
We usually don't have to worry about subfolder paths unless we're specifying the full URL. Most of the time, `Discourse.ajax` takes cares of appending the subfolder path :smile: